frontend/www/js/omegaup/components/arena/ContestCard.test.ts

```diff
@@ -0,0 +1,129 @@
+jest.mock('../../../../third_party/js/diff_match_patch.js');
+
+import { mount } from '@vue/test-utils';
+import type { types } from '../../api_types';
+import T from '../../lang';
+import arena_ContestCard from './ContestCard.vue';
+import { ContestTab } from './ContestListv2.vue';
+
+describe('ContestCard.vue', () => {
+  const daySeconds = 24 * 60 * 60 * 1000;
+  const today = new Date();
+  const yesterday = new Date(today.getTime() - daySeconds);
+  const tomorrow = new Date(today.getTime() + daySeconds);
+
+  const currentTab: ContestTab = ContestTab.Current;
+  const futureTab: ContestTab = ContestTab.Future;
+  const pastTab: ContestTab = ContestTab.Past;
+
+  const currentContest: types.ContestListItem = {
+    admission_mode: 'public',
+    alias: 'Current-Contest-1',
+    description: 'hello contest 1',
+    contest_id: 1,
+    contestants: 12,
+    finish_time: tomorrow,
+    last_updated: yesterday,
+    organizer: 'omegaup',
+    original_finish_time: tomorrow,
+    partial_score: false,
+    participating: true,
+    problemset_id: 1,
+    recommended: false,
+    start_time: yesterday,
+    title: 'Current Contest 1',
+    window_length: 300,
+  };
+
+  const futureContest: types.ContestListItem = {
+    admission_mode: 'public',
+    alias: 'Future-Contest-1',
+    description: 'hello contest 1',
+    contest_id: 1,
+    contestants: 12,
+    finish_time: new Date(tomorrow.getTime() + daySeconds),
+    last_updated: today,
+    organizer: 'omegaup',
+    original_finish_time: new Date(tomorrow.getTime() + daySeconds),
+    partial_score: false,
+    participating: true,
+    problemset_id: 1,
+    recommended: false,
+    start_time: tomorrow,
+    title: 'Future Contest 1',
+    window_length: 300,
+  };
+
+  const pastContest: types.ContestListItem = {
+    admission_mode: 'public',
+    alias: 'Past-Contest-1',
+    description: 'hello contest 1',
+    contest_id: 1,
+    contestants: 12,
+    finish_time: yesterday,
+    last_updated: new Date(yesterday.getTime() - daySeconds),
+    organizer: 'omegaup',
+    original_finish_time: yesterday,
+    partial_score: false,
+    participating: true,
+    problemset_id: 1,
+    recommended: false,
+    start_time: new Date(yesterday.getTime() - daySeconds),
+    title: 'Past Contest 1',
+    window_length: 300,
+  };
+
+  it('Should show the current contest card', async () => {
+    const wrapper = mount(arena_ContestCard, {
+      propsData: {
+        currentContest,
```

Comment:
`currentContest` no es una propiedad del componente `ContestCard`,la propiedad que se espera es `contest`, entonces se podría arreglar con:
```suggestion
        contest: currentContest,
```
Mismo caso para las otras dos pruebas